src/main/steps/no-response-yet/get.ts

```diff
@@ -0,0 +1,9 @@
+import { GetController } from '../../app/controller/GetController';
+
+import { generateContent } from './content';
+
+export class NoResponseYetApplicationGetController extends GetController {
+  constructor() {
+    super(__dirname + '/template.njk', generateContent);
```

Comment:
not sure what's happening here

Reply:
its just setting the controller up with the template location and content function. its either set it here or set it in routes.ts but having to import every content function and set the path for every page that isnt in the sequence file.